include/Reader/reader.h

```diff
@@ -3279,6 +3292,102 @@ class ReaderBase {
 
   static bool rdrIsMethodVirtual(uint32_t MethodAttribs);
 
+  /// \brief Return MethodName for non helper and non native Methods.
+  ///
+  /// \param method  Method handle for the target Method.
+  /// \param classNamePtr out param, Module name.
+  /// \param JitInfo The jit interface for this method
+  /// \returns MethodName.
+  // TODO sandreenko : change cases to LLILC rules or transfer to coreclr.
```

Comment:
If this TODO is still relevant, please move it to the implementation of `GetMethodName`, open and issue to track it, and add the issue number to the comment like so:

```
// Issue 1234: change cases to LLILC rules or transfer to coreclr.
```